lib/chef/provider/aws_sqs_queue.rb

```diff
@@ -4,9 +4,13 @@ class Chef::Provider::AwsSqsQueue < Chef::Provisioning::AWSDriver::AWSProvider
   provides :aws_sqs_queue
   
   def create_aws_object
+    options = AWSResource.lookup_options(new_resource.options || options, resource: new_resource)
```

Comment:
I'm a bit confused by `new_resource.options || options`.  I don't see `options` otherwise defined in this class so I assume it is being first defined here and thus will be nil at that point.  But, if it is nil there and that is OK, then why have the `||` at all.  I would have expected something like `new_resource.options || {}`.  Which probably would be even better by just having `{}` be the default value of `new_resource.options`.

Reply:
Yes will keep it like `new_resource.options || {}` as options are not used earlier.
Thank you.